apps/src/dance/Dance.js

```diff
@@ -691,3 +693,36 @@ Dance.prototype.captureThumbnailImage = function () {
     setThumbnailBlobFromCanvas(canvas);
   }
 };
+
+Dance.prototype.doAi = async function (input) {
+  const url = '/openai/chat_completion';
```

Comment:
This constant is defined in `src/aichat/constants.ts`.

Reply:
Will use it now.